classes/opponentPhaseCharge.lua

```diff
@@ -0,0 +1,66 @@
+-- The phase where it charges at you
+
+local opponentPhaseCharge = {}
+opponentPhaseCharge.__index = opponentPhaseCharge
+
+
+local isCharging = true
```

Comment:
remove from here, instead, have `self.isCharging = true` in the .new function. This way, we can have multiple charging phases if we like and they can all act independantly